### ISSUE TITLE
Brush toolbox item should reflect brush state

### ISSUE THREAD

**Issue by joshua-dean** [SenteraLLC/ulabel#187]
Steps to reproduce:
1. Launch single class demo
2. Select "Polygon" mode
3. Select "Brush" button in "Brush Tool"

Observe that the "Brush" button doesn't change to indicate it's active. Same with the "Erase" option. A physical change in the button would indicate that the way to de-activate it is to press the button again. My first usage of the brush tool, I attempted to click the "Polygon" mode to de-activate the brush, which didn't do anything. 

tl;dr
- Brush should have visible indication of being active
- Clicking back on the annotation mode should de-activate the brush

**Comment by joshua-dean** [comment]
The "vanish" button does this by adding a `locked` class with a different color. I think this is a fine solution, other than "locked" not being very descriptive of the actual state. 

"Enabled" and "active" are already CSS pseudo-classes. Maybe "brush-enabled" and "brush-eraser-enabled" for maximum verbosity?
